### PR TITLE
MGMT-4518 - add mac address format validation for static network conf…

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**
…ig API

### CODE REVIEW

File: models/mac_interface_map.go [hunk 1]
```diff
@@ -53,11 +54,34 @@ type MacInterfaceMapItems0 struct {
...
+	return nil
+}
+
+func (m *MacInterfaceMapItems0) validateMacAddress(formats strfmt.Registry) error {
+
+	if swag.IsZero(m.MacAddress) { // not required
+		return nil
+	}
+
+	if err := validate.Pattern("mac_address", "body", string(m.MacAddress), `^([0-9A-Fa-f]{2}[:]){5}([0-9A-Fa-f]{2})$`); err != nil {
```
Comment (flaper87) [inline models/mac_interface_map.go:81]: nit: the regex pattern could be defined as `const`

**Review by flaper87 — COMMENTED** [review]
/lgtm

**Comment by flaper87** [comment]
A bit late now that I have `lgtm` the PR but I think we should have used a more descriptive title and description for it :smiley:  next time

File: models/mac_interface_map.go (hunk 1 above)
Comment (masayag) [reply models/mac_interface_map.go:81]: This file is generated from the swagger.yaml file - see the first line of it:
`// Code generated by go-swagger; DO NOT EDIT.`

Comment (yevgeny-shnaidman) [reply models/mac_interface_map.go:81]: this is a swagger generated file

Comment (flaper87) [inline models/mac_interface_map.go:81]: Doh, good point! :sweat_smile:

**Pull request merged**